search-commons/src/main/java/no/unit/nva/search2/ticket/TicketQuery.java

```diff
@@ -106,12 +104,21 @@ public static TicketParameterValidator builder() {
      * @return TicketQuery (builder pattern)
      */
     public TicketQuery applyContextAndAuthorize(RequestInfo requestInfo) throws UnauthorizedException {
+        if (Objects.isNull(requestInfo.getUserName())) {
+            throw new UnauthorizedException();
+        }
+
         var organization = requestInfo.getTopLevelOrgCristinId()
             .orElse(requestInfo.getPersonAffiliation());
+        var adminAccess = getAccessRights(requestInfo);
 
-        return withFilterTicketType(validateAccessRight(requestInfo))
-            .withFilterOrganization(organization)
-            .withFilterCurrentUser(requestInfo.getUserName());
+        if (adminAccess.isEmpty()) {
+            withCurrentUser(requestInfo.getUserName())
+                .applyFilterCurrentUser();
+        } else {
+            withFilterTicketType(adminAccess.toArray(TicketType[]::new));
+        }
```

Comment:
Would people with DOI-access rights see their own support tickets? A curator would maybe not have support tickets, but just as an example.